recipe/meta.yaml

```diff
@@ -1,71 +1,129 @@
-{% set version = "1.2.2" %}
+{% set version = "1.3.1" %}
+{% set build_number = "0" %}
 
 package:
-  name: basemap
+  name: basemap-split
   version: {{ version }}
 
 source:
-  url: https://github.com/matplotlib/basemap/archive/v{{ version }}rel.tar.gz
-  sha256: 7e6ee5d03b10168862cff82bfa819df8264c04f078eac4549a22dd2631696613
+  url: https://github.com/matplotlib/basemap/archive/v{{ version }}.tar.gz
+  sha256: 8f4ed7d5736711ba2b413e5fe038d4349d35eb3f28cba836094b6cca5f0a634f
   patches:
-    - data.patch
+    # Update messages regarding installation of `basemap-data-hires`.
+    - install_hires_instructions.patch
 
 build:
-  number: 2
-  # This package isn't included to the SOW Packages list
-  skip: True  # [linux and s390x]
-  skip: True  # [win and py2k]
-  ignore_run_exports:
-    - proj
+  number: {{ build_number }}
+  skip: True  # [s390x]  Missing dependencies.
+  skip: True  # [win32]  Missing dependencies in win32 arch and not expecting they will be updated.
+  skip: True  # [py<3.8]  Dependencies require 3.8+
```

Comment:
`[py<38]` is a correct notation, see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=meta.yaml#preprocessing-selectors

Reply:
Fixed:
https://github.com/AnacondaRecipes/basemap-feedstock/blob/sb_update_v1.3.1/recipe/meta.yaml#L19